mmv1/templates/terraform/examples/edgenetwork_router_minimal.tf.tmpl

```diff
@@ -0,0 +1,23 @@
+
+resource "google_edgenetwork_router" "{{$.PrimaryResourceId}}" {
+  router_id   = "{{index $.Vars "edgenetwork_router_id"}}"
+  location    = "{{index $.TestEnvVars "location"}}"
+  zone        = "{{index $.TestEnvVars "zone"}}"
+  description = "Example router."
```

Comment:
The minimal example should not include any optional fields (in part to verify that they are in fact optional).

Reply:
Fixed